pinax/stripe/webhooks.py

```diff
@@ -144,8 +145,34 @@ class AccountApplicationDeauthorizeWebhook(Webhook):
     name = "account.application.deauthorized"
     description = "Occurs whenever a user deauthorizes an application. Sent to the related application only."
 
+    def validate(self):
+        """
+        Specialized validation of incoming events.
+
+        We try to retrieve the event:
+         - in case of PermissionError exception, everything is perfectly normal.
+           It means the account has been deauthorized.
+         - In case no exception has been caught, then, most likely, the event has been forged
+           to make you believe the account has been disabled despite it is still functioning.
+        """
+        stripe_account_id = self.event.webhook_message["account"]
+        self.stripe_account = models.Account.objects.filter(stripe_id=stripe_account_id).first()
```

Comment:
`.get()`?

Reply:
Do we want to blow up in case we are not in sync ?

Reply:
in lock8/pinax-stripe#35